package-lock.json

```diff
@@ -31,34 +31,10 @@
         "classnames": "2.2.5",
         "font-awesome": "4.7.0",
         "prop-types": "15.6.0",
-        "react": "16.2.0",
```

Comment:
There shouldn't be `package-lock.json` changes right?

![question](https://media.giphy.com/media/cMVgEhDeKzPwI/giphy.gif)